calculator_proj.rb

```diff
@@ -0,0 +1,42 @@
+puts "Welcome to the calculator"
+puts
+
+puts "The calculator can add,subtract,divide and multiply."
+
+puts "Please select your choice "
+
+choice = gets.chomp
+if choice == "add" || choice == "+"
+  puts "You chose addition"
+elsif
+  choice == "subtract" || choice == "-"
+  puts "You chose subtraction"
+elsif
+  choice == "multiply" || choice == "*"
+  puts " You chose multiplication"
+elsif
+  choice == "divide" || choice == "/"
+  puts "You chose division"
+else
+  puts " Calculator does not accept that input."
```

Comment:
Instead of just an `if` could you wrap this in a loop, until they entered a choice on the list?  We did something similar in the textbook notes complex conditionals.